SwiftUI_DZ/SwiftUI_DZ/ContentView.swift

```diff
@@ -6,12 +6,164 @@
 //
 
 import SwiftUI
-import CoreData
 
 struct ContentView: View {
     
+    private enum Constants {
+        static let toggleText = "Показать все счета"
+        static let balanceText = "Баланс:"
+        static let replenishedBalanceText = "Пополнить баланс"
+        static let actionSheetReplenishedText = "Баланс счета успешно пополнен"
+        static let okButtonText = "OK"
+        static let count = 500_000
+        static let openAccountText = "Открыть счет"
+        static let transferConstant = 200_000
+        static let transferButtonText = "Перевести со счета на счет"
+        static let transferAlertText = "Подтвердите перевод средств с карты на карту"
+        static let xOffsetCountPlus: CGFloat = 200
+        static let xOffsetCountMinus: CGFloat = 0
+        static let orangeViewCornerRadius: CGFloat = 20
+        static let spacerHeight: CGFloat = 200
+        static let VStackSpacer: CGFloat = 20
+        static let firstAccountText = "Счет 1:"
+        static let secondAccountText = "Счет 2:"
+        static let balanceTextCornerRadius: CGFloat = 10
+        static let toggleTextSize: CGFloat = 20
+    }
+    
+    @State private var isOn = false
+    @State private var isReplenished = false
```

Comment:
```suggestion
    @State private var isReplenishedShown = false
```
Если  флаг  что-то показывает , начинаем на is заканчиваем Shown